co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureRegistrationReceiver.java

```diff
@@ -38,9 +38,9 @@
 public class InfrastructureRegistrationReceiver implements Runnable {
     private Queue<InfrastructureRegistrationMessage> rxQueue = new LinkedList<>();
     private DatagramSocket listenSocket = null;
-    private static final int listenPort = 1515;
+    private static final int listenPort = 1615;
```

Comment:
Why are we changing this port?

Reply:
Also can we add a log statement to this class when a registration message is received?

Reply:
port being change is because it is used by [CARMARegistrationReceiver](https://github.com/usdot-fhwa-stol/carma-simulation/blob/064633572e6ce9fe08747242b9d9765ca8ad3645/co-simulation/fed/mosaic-carma/src/main/java/org/eclipse/mosaic/fed/carma/ambassador/CarmaRegistrationReceiver.java#L39)